openapi/presalytics/openapi.yml

```diff
@@ -0,0 +1,9398 @@
+openapi: 3.0.1
+servers:
+  - url: https://api.presalytics.io/ooxml-automation
+info:
+  description: |
+    This is a generated connector for [Presalytics API v1](https://presalytics.io/docs/api-specifications) OpenAPI specification.  
+    The Presalytics API provides the capability to automate reports and slide decks creation.
+  title: OOXML Automation
+  version: 0.1.0
+  x-apisguru-categories:
+    - analytics
+  x-logo:
+    url: https://presalytics.io/static/img/Logo/Logos/Orange-White/navbar.png
+  x-origin:
+    - format: openapi
+      url: https://api.presalytics.io/ooxml-automation/docs/v1/openapi.json
+      version: "3.0"
+  x-providerName: presalytics.io
+  x-serviceName: ooxml
+  x-init-description: |
+    The connector initialization requires setting the API credentials. 
+    Create a [Presalytics account](https://presalytics.io/signup) and obtain tokens by following [this guide](https://presalytics.io/docs/api-specifications/).
```

Comment:
I couldn't find the `securityScheme:` in this openapi. Could you please check again?

Reply:
yes. `securityScheme` is not included in the openapi specification. I tried to check the API document, but that was down last week and today whole website is down  https://presalytics.io/